src/Serialize.h

```diff
@@ -33,6 +33,101 @@ namespace drafter {
 
     refract::Registry& GetNamedTypesRegistry();
 
+    template<typename T>
+    struct SectionInfo {
+        typedef SectionInfo<T> Type;
+        typedef T SectionType;
+        typedef snowcrash::SourceMap<SectionType> SourceMapType;
```

Comment:
`typedef snowcrash::SourceMap<T> SourceMapType;`

I know it's the same but this would be more easier to read and understand for other people.